test/prob/test_fixture_cdf.hpp

```diff
@@ -3,6 +3,7 @@
 
 #include <stan/math/rev.hpp>
 #include <test/prob/utility.hpp>
+#include <test/unit/math/expect_near_rel.hpp>
```

Comment:
All the changes in this file are similar to the equivalent ones in the lccdf file.